src/targets/mobile/containers/onboarding/Files.jsx

```diff
@@ -4,7 +4,7 @@ import OnBoarding from '../../components/OnBoarding'
 
 export const Files = ({ onActivate }) =>
 (
-  <OnBoarding onActivate={onActivate} stepName='files' currentStep={1} totalSteps={3} />
+  <OnBoarding onActivate={onActivate} stepName='files' currentStep={1} totalSteps={4} />
```

Comment:
I know that this magic number did not appear in your PR, but can we find a way to avoid this?
I guess it is used for the breadcrumb.